app/jenkins-for-jira-ui/src/components/ConnectionPanel/NotConnectedState.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { cx } from '@emotion/css';
+import Button from '@atlaskit/button';
+import { ConnectedState } from '../StatusLabel/StatusLabel';
+import {
+	notConnectedStateContainer,
+	notConnectedStateHeader,
+	notConnectedStateParagraph, notConnectedTempImgPlaceholder
+} from './ConnectionPanel.styles';
+
+type NotConnectedStateProps = {
+	connectedState: ConnectedState
+};
+
+const NotConnectedState = ({ connectedState }: NotConnectedStateProps): JSX.Element => {
+	const notConnectedHeader =
+        connectedState === ConnectedState.PENDING ? 'Connection pending' : 'Duplicate server';
```

Comment:
/nit It'd be good to place this set of ternaries on a new line like you've done with the others to make it more readable.

```
const notConnectedHeader = connectedState === ConnectedState.PENDING 
    ? 'Connection pending' 
    : 'Duplicate server';
```